lib/inertia_rails.rb

```diff
@@ -20,3 +20,12 @@
 module InertiaRails
   class Error < StandardError; end
 end
+
+class ActionDispatch::Routing::Mapper
```

Comment:
is this the best practice method for adding methods to Rails classes?

this is a genuine question... i know there are several ways to extend classes in Ruby.

Reply:
Good call second-guessing this. It looks like doing it this way in a gem creates a risk that if things are executed in the wrong order, we could end up defining that class before rails. 

Luckily we already have a pattern for doing this in our gem that I missed so I changed this to mimic that.